### PR TITLE
Local auth app to radix namespace

### PR DESCRIPTION
**Pull request by ElijahAhianyo**

### CODE REVIEW

File: local_auth/local_auth/local_auth.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+
+def index() -> rx.Component:
+    """Render the index page.
+
+    Returns:
+        A reflex component.
+    """
+    return rx.fragment(
+        rx.chakra.color_mode_button(rx.chakra.color_mode_icon(), float="right"),
```
Comment (martinxu9) [inline local_auth/local_auth/local_auth.py:16]: I think there's a new color mode button?

File: local_auth/local_auth/login.py [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+def login_page() -> rx.Component:
+    """Render the login page.
+
+    Returns:
+        A reflex component.
+    """
+
+    login_form = rx.box(
+        rx.vstack(
+            rx.form.root(
```
Comment (martinxu9) [inline local_auth/local_auth/login.py:72]: we can use `rx.form` instead of `rx.form.root` now

File: local_auth/local_auth/login.py [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+                    rx.heading(
+                        "Login into your Account", size="7", margin_bottom="2rem"
+                    ),
+                    rx.text(
+                        "Username",
+                        color="hsl(240, 5%, 64.9%)",
+                        margin_top="2px",
+                        margin_bottom="4px",
+                    ),
+                    rx.input.input(
```
Comment (martinxu9) [inline local_auth/local_auth/login.py:83]: should we use `rx.input` instead? there's another one below on line 95

File: local_auth/local_auth/login.py [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+                        placeholder="password",
+                        id="password",
+                        border_color="hsl(240,3.7%,15.9%)",
+                        justify_content="center",
+                        type="password",
+                    ),
+                    rx.box(
+                        rx.form.submit(
```
Comment (martinxu9) [inline local_auth/local_auth/login.py:103]: we could get rid of the `rx.form.submit` here and use the button directly.

File: local_auth/local_auth/registration.py [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+def registration_page() -> rx.Component:
+    """Render the registration page.
+
+    Returns:
+        A reflex component.
+    """
+
+    register_form = rx.box(
+        rx.vstack(
+            rx.form.root(
```
Comment (martinxu9) [inline local_auth/local_auth/registration.py:84]: had similar comments about `rx.form.root`, `rx.input.input` in login.py

File: local_auth/requirements.txt [hunk 6]
```diff
@@ -0,0 +1,3 @@
+reflex>=0.3.8
```
Comment (martinxu9) [inline local_auth/requirements.txt:1]: should we say >=0.4.0 here?

File: local_auth/rxconfig.py [hunk 7]
```diff
@@ -0,0 +1,8 @@
+import reflex as rx
+
+class LocalauthConfig(rx.Config):
+    pass
+
+config = LocalauthConfig(
```
Comment (martinxu9) [inline local_auth/rxconfig.py:6]: should we make it simpler? `config = rx.Config(app_name="local_auth")` instead?

**Review by martinxu9 — CHANGES_REQUESTED** [review]

**Review by masenf — APPROVED** [review]

**Pull request merged**